static/js/admin/src/ResultObjectActionMenu.js

```diff
@@ -205,7 +205,12 @@ define('ResultObjectActionMenu', [ 'jquery', 'jquery-ui', 'StringUtilities',  'A
 		}
 
 		if (metadata.type === 'File' && $.inArray('ingest', metadata.permissions) !== -1) {
-			items["editStreamingProperties"] = {name : 'Edit Streaming Properties'};
+			items["streaming"] = {name: "Streaming Properties", items: {}}
+			items["streaming"]['items']["editStreamingProperties"] = {name: "Edit Streaming URL"};
```

Comment:
Might make sense to keep it as Edit Streaming Properties and Delete Streaming Properties, since we're going to be adding another property shortly that will be edited at the same time.